### DISCUSSION TITLE
Graph to show Daily/Weekly/Monthly Mileage

### DISCUSSION THREAD

**Discussion by natrlhy** [teslamate-org/teslamate#724] [Ideas]
I can't find it, but somewhere on reddit, I saw someone post a nice graph that displayed how many miles on average they weren't driving due to COVID-19 stay at home, etc.

It got me wishing that there was a graph to look at:

Miles driven today
Miles driven this week
Miles driven this month
Miles driven this year
etc...

Basically a way to break down graphically drive patterns and how the fluctuate visually.
It would help display things like, "I drive more on Mondays" or "I now see I tend to drive less now that I work from home more" or "Boy, I sure run a lot of errands on the weekend!". Things like that.

I wasn't sure if this exists already so please pardon if this has been asked or addressed already.
Thanks!

**Comment by baylanger** [comment]
"Boy, I sure run a lot of errands on the weekend!" - do we want on this dashboard the "number of trips / day,week,month,year"?  Maybe we could configure a value that defines anything below X minutes is considered a errands and have a counter for trips and errands :)

Your request made me think of another dashboard.  Would it be great to have a dashboard that allows to use the geo-fence defined locations (home, work, cottage, etc) and visualize trips between 2 defined locations?  For example home-to-work or work-to-work, etc. The dashboard would show stats between the 2 locations.  Min/max/avg Time to travel, consumption, etc. Perhaps it's even possible to highlight on the map in green/yellow/orange/red based on traffic or not.  Seems this info would be great to see for pre/during and post covid period.  How long will it take for traffic to be as bad as pre-covid?  🤔 I'll likely open a request for this.

**Comment by 3rdStng** [comment]
Both are great dashboard ideas!
